### DISCUSSION TITLE
Enquiry regarding persistent worker

### DISCUSSION THREAD

**Discussion by ghost** [cirruslabs/cirrus-ci-docs#1240] [Q&A]
Hola there!
I have installed persistent worker on my host machine so that the ci runs the triggered builds on my machine. But I have some questions regarding this. 
a) Since the time limit for free users is 2 hours, so can I exceed the limit and define it as >2hr in the .cirrus.yml script as I am running the build on persistent worker not on the community container ?
b) The resources limit for free users is 32G ram and 8 cpu. Take for instance, my machine has 64 gigs of ram and 32 cpu. So can I exceed the limit and set the ram to 64G and cpu to 32 in .cirrus.yml script ?

Answering would help me a lot.
Thanking you in advance

**Comment by fkorotkov** [comment]
You should be able to use more than 8 CPUs and timeouts greater than 2 hours.

**Reply by ghost** [reply]
Thank you sir a lot. I have been using Cirrus CI from months and I am satisfied with the services you guys provide and that too for free. The timeout on community containers is 2 hr. It's my  request you to increase the timeout to 30 minutes more. That would do me and my organisation a lot of help as being students, we don't have enough funds to run our builds on our machines and depend on you guys. Therefore I and my team would be very grateful to Cirrus CI if the timeout limit is increased. Again thank you for all the things/services you have provided us

Thanks
